schema/sale.js

```diff
@@ -10,6 +12,18 @@ import {
   GraphQLBoolean,
 } from 'graphql';
 
+export function auctionState({ start_at, end_at }) {
```

Comment:
Is `auction_state` from Gravity unreliable, or is this a slightly different definition of `auction_state`?